integrations/connext/components/transfer.tsx

```diff
@@ -0,0 +1,168 @@
+import moment from 'moment'
+import Image from 'next/image'
+import { BsInfoCircle } from 'react-icons/bs'
+import { HiOutlineCheckCircle } from 'react-icons/hi'
+import { IoWarning } from 'react-icons/io5'
+import { TiArrowRight } from 'react-icons/ti'
+
+import { mainnetAssets, testnetAssets } from '../utils/assets'
+import { mainnetChains, testnetChains } from '../utils/chains'
+import { Asset, Chain, Transfer } from '../utils/types'
+
+type TransferProps = {
+  isMainnet: boolean
+  transfer: Transfer
+}
+
+function formatTimestamp(timestamp: number): string {
+  // eslint-disable-next-line
+  const formattedTimestamp: string = moment(timestamp * 1000).format('MMM D, YYYY h:mm:ss A')
+  return formattedTimestamp
+}
+
+function findChain(chains: Chain[], domainId: string): Chain | undefined {
+  return chains.find((chain) => chain.domain_id === domainId)
+}
+
+function findAsset(assets: Asset[], transactingAsset: string): Asset | undefined {
```

Comment:
these helper functions could be moved to a utils folder to improve the readability, but up to you